sdk/resourcemanager/azure-resourcemanager-containerinstance/pom.xml

```diff
@@ -45,51 +50,55 @@
+            <version>2.0.0-beta.4</version>  <!-- {x-version-update;com.azure.resourcemanager:azure-resourcemanager-resources;current} -->
         </dependency>
         <dependency>
             <groupId>com.azure.resourcemanager</groupId>
             <artifactId>azure-resourcemanager-storage</artifactId>
-            <version>2.0.0-SNAPSHOT</version>
+            <version>2.0.0-beta.4</version>  <!-- {x-version-update;com.azure.resourcemanager:azure-resourcemanager-storage;current} -->
         </dependency>
         <dependency>
             <groupId>com.azure.resourcemanager</groupId>
             <artifactId>azure-resourcemanager-authorization</artifactId>
-            <version>2.0.0-SNAPSHOT</version>
+            <version>2.0.0-beta.4</version>  <!-- {x-version-update;com.azure.resourcemanager:azure-resourcemanager-authorization;current} -->
         </dependency>
         <dependency>
             <groupId>com.azure.resourcemanager</groupId>
             <artifactId>azure-resourcemanager-msi</artifactId>
-            <version>2.0.0-SNAPSHOT</version>
+            <version>2.0.0-beta.4</version>  <!-- {x-version-update;com.azure.resourcemanager:azure-resourcemanager-msi;current} -->
         </dependency>
         <dependency>
             <groupId>com.azure.resourcemanager</groupId>
             <artifactId>azure-resourcemanager-network</artifactId>
-            <version>2.0.0-SNAPSHOT</version>
+            <version>2.0.0-beta.4</version>  <!-- {x-version-update;com.azure.resourcemanager:azure-resourcemanager-network;current} -->
         </dependency>
         <dependency>
             <groupId>com.azure</groupId>
             <artifactId>azure-storage-file-share</artifactId>
-            <version>12.5.0</version>
+            <version>12.6.0-beta.1</version>  <!-- {x-version-update;com.azure:azure-storage-file-share;dependency} -->
         </dependency>
         <dependency>
           <groupId>org.junit.jupiter</groupId>
           <artifactId>junit-jupiter-engine</artifactId>
+          <version>5.6.2</version> <!-- {x-version-update;org.junit.jupiter:junit-jupiter-engine;external_dependency} -->
           <scope>test</scope>
         </dependency>
         <dependency>
           <groupId>org.junit.jupiter</groupId>
           <artifactId>junit-jupiter-api</artifactId>
+          <version>5.6.2</version> <!-- {x-version-update;org.junit.jupiter:junit-jupiter-api;external_dependency} -->
           <scope>test</scope>
         </dependency>
         <dependency>
           <groupId>com.azure</groupId>
           <artifactId>azure-core-http-netty</artifactId>
+          <version>1.5.3</version> <!-- {x-version-update;com.azure:azure-core-http-netty;dependency} -->
           <scope>test</scope>
         </dependency>
         <dependency>
           <groupId>com.azure</groupId>
           <artifactId>azure-identity</artifactId>
+          <version>1.0.8</version> <!-- {x-version-update;com.azure:azure-identity;dependency} -->
```

Comment:
azure-identity appears to be 1.0.9?

Reply:
Just copy from another pom, I think they could be update later with the script.